builds/misc/images-release.yaml

```diff
@@ -386,11 +386,9 @@ jobs:
 ################################################################################
     displayName: Windows
     pool:
-      name: Azure-IoT-Edge-Core
+      name: $(pool.windows.name)
       demands: 
-        - Build-Image -equals true
-        - win-rs5
-        - DotNetFramework
+       - ImageOverride -equals agent-aziotedge-winserver-2019dc-build
```

Comment:
Please use Windows Enterprise LST instead of Windows Server. 
`agent-aziotedge-winserver-2019dc-build` -> `agent-aziotedge-win10-entn2019-build`

Reply:
I tried that image initially but the build failed because it did not have Java installed on it. Here's the pipeline run from that attempt: https://msazure.visualstudio.com/One/_build/results?buildId=46366280&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=16be6190-c20b-50b6-8210-aa45e5159981

Reply:
Windows Server 2019 is locked to Windows 10.0.17763, just like Windows 10 LTS. `agent-aziotedge-winserver-2019dc-build` is the right image to use.